config/config.go

```diff
@@ -1438,7 +1438,8 @@ type LimitRange struct {
 
 type NetworkPolicy struct {
 	// Enabled defines if the network policy should be deployed by vCluster.
-	Enabled bool `json:"enabled,omitempty"`
+	Enabled         bool `json:"enabled,omitempty"`
+	PlatformEnabled bool `json:"platformEnabled,omitempty"`
```

Comment:
Platform enabled is a little bit misleading, can we move this into `outgoingConnections.platform: true`?

Reply:
I followed what was in the ticket